<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="9" column="0">
     <widget class="QTextEdit" name="te_info">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Сигнал 2:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_filename2">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="openFile2">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCustomPlot" name="risunok" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>180</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Частота дискретизации</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_fd">
        <property name="text">
         <string>800000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Размер считваемых фрагментов</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_razm">
       <property name="text">
        <string>74850</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Сигнал 1:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_filename1">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="openFile1">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pb_VKF_exec">
      <property name="text">
       <string>Посчитать ВКФ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
